modules/signoz/README.md

```diff
@@ -0,0 +1,62 @@
+# Purpose
+The purpose of this module is to deploy the `Signoz` helm chart <https://github.com/SigNoz/charts/tree/main/charts/signoz>.
+
+SigNoz is an open-source APM. It helps developers monitor their applications 
+& troubleshoot problems, an open-source alternative to DataDog, NewRelic, etc. Open 
+source Application Performance Monitoring (APM) & Observability tool.
+
+
+## This module is a work in progress
+A number of items are needed:
+
+- Setting up backups and data retention: https://sagebionetworks.jira.com/browse/IBCDPE-1094
+- Set up ingress to the cluster/collector to send data to: https://sagebionetworks.jira.com/browse/IBCDPE-1095
+- Set up accounts and access to the service decleratively
+
+## Accessing signoz
```

Comment:
SageIT would be very interested to have this setup with SSO+SCIM to help with onboarding/offboard users, otherwise unauthorized user access will be inevitable.

Reply:
My thought was to use the Synapse OIDC flow for users accessing the UI. I need to further document me exact plan but the idea was:

1. Create a client to authenticate users with Synapse
2. Verify the users membership to a specific Synapse group
3. As users are added/removed from the Synapse group they will gain/lose access to the UI

Signoz self-hosted doesn't integrate with SSO. So what we would have to do is have users log in through Synapse oidc (Which IT could help control), and then further log in through the Signoz user interface to their specific account.

Thoughts?